README.md

```diff
@@ -1,6 +1,51 @@
 # Northcoders News API
 
-Please create enviroment variables to be able to clone and locally run this file succesfully. Please create the following files:
+This is an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture. It accesses different databases (articles, comments, topics, and users) and uses these to create multiple end points each with a specific use, detailed in get/API.
```

Comment:
A quick clarification: articles, comments, topics, users are **tables**, not databases.

Something in here that clarifies what sort of data we are serving would be good! Just a "this API serves JSON data for a news website, consisting of articles, comments, topics, and users info."